contrib/scripts/jenkins-failures.sh

```diff
@@ -3,9 +3,12 @@
 # This script checks Jenkins master Jobs and validated that the failed jobs has
 # been already triaged and upstream issue has been created if it's needed.
 # To clean the list of Jenkins builds, build description needs to be updated with a comment.
-NOW=$(printf "%(%s)T" -1);
-YESTERDAY=$(printf "%(%s)T\n" $((NOW - 24*3600)))
-FROM=$(date -d @$YESTERDAY +%s%N  | cut -b1-13)
+
+# allow users to specify SINCE as seconds since epoch
+YESTERDAY=$(date -d -1day +%s)
+SINCE=${SINCE:-$YESTERDAY}
```

Comment:
Love this one :-)

Reply:
It worked out much better than I expected, actually :)